src/pages/learn.js

```diff
@@ -65,12 +69,19 @@ const HeaderContent = styled('div')`
     margin-right: auto;
 `;
 
-const Article = styled('article')`
+const CardContainer = styled('div')`
```

Comment:
Can we undo this? using `article` is a good practice for a main piece of content